cmake-init/templates/common/BUILDING.md

```diff
@@ -79,6 +79,37 @@ target_link_libraries(
     {= name =}::{= name =}
 )
 ```
+
+### Creating a VCPKG port
```

Comment:
"Officially" `vcpkg` is used like a regular word in its docs, so it shouldn't be all caps.

The additional directory in the include path also affects every other form of packaging, there is no need to make it vcpkg specific. Something like a "Note to packagers" section with cautioning them to review the usage of `CMAKE_INSTALL_INCLUDEDIR`. This should also be guarded with `{% if not exe %}`.